indico/modules/events/editing/service.py

```diff
@@ -148,3 +135,61 @@ def service_handle_new_editable(editable):
     except requests.RequestException as exc:
         logger.exception('Failed calling listener for editable')
         raise ServiceRequestFailed(exc)
+
+
+def service_handle_review_editable(editable, action, parent_revision, revision=None):
+    new_revision = revision or parent_revision
+    data = {
+        'action': action.name,
+        'revision': EditingRevisionUnclaimedSchema().dump(new_revision),
+        'endpoints': _get_revision_endpoints(new_revision)
+    }
+    try:
+        path = '/event/{}/editable/{}/{}/{}'.format(
+            _get_event_identifier(editable.event),
+            editable.type.name,
+            editable.contribution_id,
+            new_revision.id
+        )
+        resp = requests.post(_build_url(editable.event, path), headers=_get_headers(editable.event),
+                             json=data)
+        resp.raise_for_status()
+        resp = ServiceReviewEditableSchema().load(resp.json())
```

Comment:
This schema also contains a `comments` field, but this is not being handled anywhere. Also see my related TODO comment in `process_revision` in indico/openreferee#5

Reply:
I had some changes stashed for this. But ended up concluding we don't actually need it, after all, using the endpoint works!

Reply:
not if the revision does not exist yet...